cmd/dispatcher/main.go

```diff
@@ -74,6 +78,17 @@ func main() {
 		logger.Fatalf("Failed to build coordination client: %s", err.Error())
 	}
 
+	redisPW, err := ioutil.ReadFile(o.redisPW)
+	if err != nil { //wasnt able to find the path to the file/not sure where it is
+		logger.Fatalf("Failed to retrieve database password from a file: %s", err.Error())
+	}
+
+	redisClient := redis.NewClient(&redis.Options{
+		Addr:     o.redisURL,
+		Password: string(redisPW),
```

Comment:
Lets make sure that there are no extra spaces, lines when we get the data from the file. Check https://godoc.org/strings#TrimSpace